### DISCUSSION TITLE
Add Claude (or other AIs) to your script?

### DISCUSSION THREAD

**Discussion by michaelbeijer** [kdalanon/ChatGPT-AutoHotkey-Utility#8] [Ideas]
Hi @kdalanon!

It would be super cool if we could also use Claude 3 Opus in the script. Is there any way you could add this? I am very happily using your script inside [my own script](https://beijer.bot/), but have recently started using Claude 3 Opus, and it is amazing.

Michael

**Comment by kdalanon** [comment]
Hi @michaelbeijer,

Thank you for using my script, I really appreciate it!

I'm considering integrating additional AI APIs into the script, although there are no immediate plans to do so.

**Reply by michaelbeijer** [reply]
Thanks!

I would really love to use Gemini!!!
